Extensions/TweenBehavior/JsExtension.js

```diff
@@ -0,0 +1,532 @@
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectScaleTween");
+
+    behavior
+      .addAction(
+        "AddObjectScaleXTween",
+        _("Add object X-scale tween"),
+        _(
+          "Add a tween animation for the object X-scale. (Note: the scale can never be less than 0)"
+        ),
+        _(
+          "Tween the X-scale of _PARAM0_ to _PARAM3_ with easing _PARAM4_ over _PARAM5_ms as _PARAM2_"
+        ),
+        _("Scale"),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To scale X"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), easingChoices, false)
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectScaleXTween");
+
+    behavior
+      .addAction(
+        "AddObjectScaleYTween",
+        _("Add object Y-scale tween"),
+        _(
+          "Add a tween animation for the object Y-scale. (Note: the scale can never be less than 0)"
+        ),
+        _(
+          "Tween the Y-scale of _PARAM0_ to _PARAM3_ with easing _PARAM4_ over _PARAM5_ms as _PARAM2_"
+        ),
+        _("Scale"),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To scale Y"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), easingChoices, false)
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectScaleYTween");
+
+      behavior
+      .addAction(
+        "AddTextObjectCharacterSizeTween",
+        _("Add text size tween"),
+        _(
+          "Add a tween animation for the text object character size. (Note: the size can never be less than 1)"
```

Comment:
Same in the rest of the file if you can change how parenthesis are written :)
I know it's a pain but that's what make the difference between a professional looking software and an amateur one for native English speakers.